### DISCUSSION TITLE
CI workflow fails on Ruby 1.9.3 due rails-html-sanitizer 1.5.0 release

### DISCUSSION THREAD

**Discussion by v-kolesnikov** [jeremyevans/forme#112] [General]
I've noticed a fail on #111 and run workflow agains master branch and got the same issue:
https://github.com/v-kolesnikov/forme/actions/runs/4023984568/jobs/6915486549#step:4:15

Quick glance lead me here: https://github.com/rails/rails-html-sanitizer/compare/v1.4.4...v1.5.0#diff-0daf33b9062eb5ccdeae86ed8bf2662a6e8669f1a7db590802b7f3b36ea47426R113

rails-html-sanitizer was released recently as 1.5.0 that is incompatible with Ruby 1.9.3, but supported Ruby version is not specified in this gem.

Possible solutions:

- Specify rails-html-sanitizer version manually for CI workflow as 1.4.4 for Ruby 1.9.3 by creating a custom Gemfile
- Do not run Rails integration test on Ruby 1.9.3
- Stop support Ruby 1.9.3 for Forme

Ready to help with this.

**Comment by jeremyevans** [comment]
This type of issue pops up occasionally for older Ruby versions. I generally work around it by specifying gem version constraints based on the Ruby version.  I'll take care of this after merging #111.
